db/seeds.rb

```diff
@@ -0,0 +1,40 @@
+users = 5.times.map do |x|
```

Comment:
What about to pick 1 main user and print out credentials to the console to make it simple login after seed is "planted"?

Reply:
could do that, or what I've been doing in the console is `u = User.first; u.password = "password"; u.save!(validate: false)`

Reply:
Currently it is impossible to get the login credentials after running seeds. There is need for this additional step. I'll open new PR.